flow/table.go

```diff
@@ -368,6 +368,7 @@ func (ft *Table) packetToFlow(packet *Packet, parentUUID string) *Flow {
 		}
 
 		flow.Update(packet, ft.flowOpts)
+		flow.newApplicationLayer(packet, ft.flowOpts)
```

Comment:
This is not a new flow. I would prefer that you refactor your code in order to trigger the DNS parsing outside of the newApplicationLayer function. So that you could call it from the newApplicationLayer and from the update if needed

Reply:
done